src/main/shell-session/local-shell-session.ts

```diff
@@ -32,12 +32,11 @@ export class LocalShellSession extends ShellSession {
   }
 
   public async open() {
-
     const env = await this.getCachedShellEnv();
-    const shell = env.PTYSHELL;
+    const shell = env.SHELL || env.PTYSHELL;
```

Comment:
`this.getCachedShellEnv()` is setting `env.PTYSHELL` based on user preferences (with fallback to `env.SHELL`). I don't see any reason to modify it again here. Could you describe what bug this actually fixes?

Reply:
You are right. I had it set...